compiler/record-minmax/src/MinMaxObserver.cpp

```diff
@@ -51,6 +51,16 @@ void MinMaxObserver::postTensorWrite(const luci::CircleNode *node,
     // Bool type tensor is not quantized
     return;
   }
+  if (node->dtype() == DataType::S32)
+  {
+    // Integer type tensor is not quantized
+    return;
+  }
+  if (node->dtype() == DataType::S64)
+  {
+    // Integer type tensor is not quantized
+    return;
+  }
```

Comment:
I see that this code is already merged, just want to ask how this code is interacts with code below?
```
if (tensor->element_type() != DataType::FLOAT32)
```

Reply:
Personally, I think it's OK to skip quantization for all integer tensors and just return, when observer detects node with integer type. But as I understand now, this code makes `switch`/`case` statement bellow redundant in most cases. So, I think better to ask some advice about correctness of this approach.

@jinevening Could you please take a look? =)

Reply:
> But as I understand now, this code makes switch/case statement bellow redundant in most cases

Yes, this is what attracted my attention =)

Reply:
record-minmax records min/max of fp32 feature maps. AFAIK, int32 and int64 are not used for feature maps, so I approved the change (@binarman please tell me if I missed something).

It seems that the case for ARG_MAX becomes redundant in the below switch statement. @m-bronnikov could you remove the case?

Reply:
> int32 and int64 are not used for feature maps

So far I saw integer operations only for indexing, so you are right.